apps/src/types/utils.ts

```diff
@@ -14,3 +14,12 @@ export function convertOptionalStringToBoolean(
     return value === 'true';
   }
 }
+
+/**
+ * Return the keys of an object as a typed array.
+ */
+export function getTypedKeys<K extends string | number | symbol>(object: {
```

Comment:
Added this TypeScript util so we don't have to manually cast keys whenever iterating through the keys of an interface (@molly-moen lmk if you think this should go somewhere else, I chose this file a bit arbitrarily).

Reply:
this seems as good a place as any!